test/apiv2/10-images.at

```diff
@@ -171,84 +171,32 @@ function cleanBuildTest() {
 }
 CONTAINERFILE_TAR="${TMPD}/containerfile.tar"
 cat > $TMPD/containerfile << EOF
-FROM quay.io/libpod/alpine_labels:latest
+FROM $IMAGE
 EOF
 tar --format=posix -C $TMPD -cvf ${CONTAINERFILE_TAR} containerfile &> /dev/null
 
-curl -XPOST --data-binary @<(cat $CONTAINERFILE_TAR) \
-    -H "content-type: application/x-tar" \
-    --dump-header "${TMPD}/headers.txt" \
-    -o "${TMPD}/response.txt" \
-    "http://$HOST:$PORT/v1.40/libpod/build?dockerfile=containerfile" &> /dev/null
-
-BUILD_TEST_ERROR=""
-
-if ! grep -q '200 OK' "${TMPD}/headers.txt"; then
-    echo -e "${red}NOK: Image build from tar failed response was not 200 OK (application/x-tar)"
-    BUILD_TEST_ERROR="1"
-fi
-
-if ! grep -q 'quay.io/libpod/alpine_labels' "${TMPD}/response.txt"; then
-    echo -e "${red}NOK: Image build from tar failed image name not in response"
-    BUILD_TEST_ERROR="1"
-fi
-
-curl -XPOST --data-binary @<(cat $CONTAINERFILE_TAR) \
-    -H "content-type: application/tar" \
-    --dump-header "${TMPD}/headers.txt" \
-    -o "${TMPD}/response.txt" \
-    "http://$HOST:$PORT/v1.40/build?dockerfile=containerfile&q=true" &> /dev/null
-if ! grep -q '200 OK' "${TMPD}/headers.txt"; then
-    echo -e "${red}NOK: Image build from tar failed response was not 200 OK (application/tar)"
-    BUILD_TEST_ERROR="1"
-fi
-if grep -E "\"[0-9a-f]{64}\\\n\"" $(jq .stream "${TMPD}/response.txt"); then
```

Comment:
This actually never worked. All it did was throw:
```
grep: "4fab981df737963d83b548a3a6f4e7b43f718eb88567ba5ec7921f636a93d7bc\n": No such file or directory
```
...because I think the intent was to check the output of `jq.stream.etc`, not treat that output as a file. So this test was actually not doing anything.

Reply:
I think was doing the right thing; I spent some time to get (what I believed) was right.

It was meant to make sure that the only data in the stream is `$ID\n`.

Reply:
But I like the current solution _much more_, thanks!

Reply:
@vrothberg I know that was the intent, but it actually didn't work. To simplify the description of the problem, your intention was: _check that CID matches PATTERN_ but what was happening instead was:
```console
$ grep -E PATTERN $(jq .stream FILE)
= grep -E PATTERN CID
```
...which tries to find PATTERN in a file called CID which throws the above ENOENT error. Look in any log file of the APIv2 tests in the past few months, and you will see all of them (as many as you wish to check) throwing that ENOENT error. The test was a NOP. And because of my carelessness in not doing the fail-on-error logic earlier, the error was silently being ignored.

Reply:
Thanks for the explanation! That's curious since the check passed only after the patch, and I recall that it failed until I got the quoting `\\\n` right.